### ISSUE TITLE
Some images a Served with low resolution

### ISSUE THREAD

**Issue by jlvn** [theoaktells/theoaktells.com#23]
Image natural dimensions should be proportional to the display size and the pixel ratio to maximise image clarity. [Learn how to provide responsive images](https://web.dev/articles/serve-responsive-images?utm_source=lighthouse&utm_medium=lr).
